buku

```diff
@@ -466,6 +466,16 @@ class BukuDb:
             except IndexError:
                 print('Index out of bound')
 
+    def get_bookmark_by_id(self, index):
+        """Get a bookmark from database by its ID.
+        Return data as a tuple. Return None if index not found.
+        """
+
+        self.cur.execute('SELECT * FROM bookmarks WHERE id = ?', (index,))
+        results = self.cur.fetchall()
+        if len(results) > 0:
```

Comment:
Otherwise, can we explicitly return `None`?
It's better to have it complete in the first shot.